v21.1/restore.md

```diff
@@ -157,8 +151,8 @@ You can either restore from a full backup or from a full backup with incremental
 
 Restore Type | Parameters
 -------------|----------
-Full backup | Include only the path to the full backup.
-Full backup + <br>incremental backups | If the full backup and incremental backups were sent to the same destination, include only the path to the full backup (e.g., `RESTORE FROM 'full_backup_location';`).<br><br>If the incremental backups were sent to a different destination from the full backup, include the path to the full backup as the first argument and the subsequent incremental backups from oldest to newest as the following arguments (e.g., `RESTORE FROM 'full_backup_location', 'incremental_location_1', 'incremental_location_2';`).
+Full backup | Include the [subdirectory](#view-the-backup-subdirectories) of the backup and the path to the full backup destination (e.g.`RESTORE ... FROM 'subdirectory' IN 'backup destination';`).
```

Comment:
I think this is a little confusing because we're providing the user with the same command in each scenario. If the subdirectory contains an incremental backup, it's going to get included in the RESTORE--the user doesn't really have a choice. @livlobo do users ever only want to restore a full backup, even though they created incrementals-- i doubt that user story exists.

Reply:
Yes, I get your point. I think without the context of the examples below and distinguishing the backup type with `SHOW BACKUP` this could get a little confusing. I was attempting to distinguish this with "backup collection", but that perhaps does not cover it. 

What about dropping the e.g. statements here. I have a feeling that it could be better to link directly to the examples for the statements in place.

Then for the Full backup + incr backup sentence, clarify that if there are incremental backups in the backup collection the restore will include it?

Reply:
I think your idea makes sense! And the revision looks good.

Reply:
Looks good.